docs/user/running-cincinnati.md

```diff
@@ -0,0 +1,71 @@
+# Running Cincinnati
+
+## Prerequisite
+
+APP-SRE publishes the Cincinnati container images at [Quay registry](https://quay.io/repository/app-sre/cincinnati)
+
+We need an image tag from the [Quay registry](https://quay.io/repository/app-sre/cincinnati) to use with the deployment config.
+
+The Cincinnati deploymentconfig present in the git repository at [dist/openshift/](https://github.com/openshift/cincinnati/blob/master/dist/openshift/cincinnati.yaml)
```

Comment:
No need for an absolute URI here, you can use `../../dist/openshift/cincinnati.yaml`).